src/pages/curriculum-page.tsx

```
@@ -0,0 +1,22 @@
+import CurriculumBox from '@/shared/components/curriculum-box/curriculum-box';
+import * as styles from '@/shared/components/curriculum-box/curriculum-box.css';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)